packages/neon-core/src/wallet/components/AssetBalance.ts

```diff
@@ -29,6 +31,8 @@ export class AssetBalance implements NeonObject {
     this.unconfirmed = abLike.unconfirmed
       ? abLike.unconfirmed.map((coin) => new Coin(coin))
       : [];
+
+    this.hash = abLike.hash || "";

```

Comment:
Once again, should we be allowing this field to be unpopulated? I would say no unless we have serious issues trying to fill this field.

Reply:
correction applied